connectors/src/connectors/github/lib/github_api.ts

```diff
@@ -221,7 +221,10 @@ export async function getRepoIssuesPage(
         "transient_upstream_activity_error"
       );
     }
-
+    // Handle disabled issues case - GitHub returns 410 Gone when issues are disabled
+    if (err instanceof RequestError && err.status === 410) {
```

Comment:
Maybe we should also add a filter on the message "Issues are disabled for this repo" to make sure we only silence errors for this specific expected case? Cause since we return empty array we completely silence the error. 

What does happen on the permission tree when issues are disabled, do we still display the "Issues" entry? Not sure we've handled this use case? Also, does anything needs to be done also for the Github Discussions?

Reply:
> Maybe we should also add a filter on the message "Issues are disabled for this repo" to make sure we only silence errors for this specific expected case? Cause since we return empty array we completely silence the error.

Looking at their docs this is the only case in which we get 410. But can definitely add a filter on the message too.

Reply:
> What does happen on the permission tree when issues are disabled, do we still display the "Issues" entry? Not sure we've handled this use case? Also, does anything needs to be done also for the Github Discussions?

Not sure either how this would be handled in the permission tree, looking now.
Regarding discussion I haven't seen any logs yet, will port the same changes if it pops up.